rust-codegen/codegen/README.md

```diff
@@ -0,0 +1,9 @@
+# kw-rs
```

Comment:
Ideally this module would have test coverage. All other languages that Kiwi generates code for have tests here: https://github.com/evanw/kiwi/tree/master/test. The tests both snapshot the generated code and also make sure that the generated code works correctly. All tests do roughly the same actions.

Can you add tests for your Rust code generator like these other tests? The tests are run using `test.sh`. I'm happy to help out writing tests.